### PR TITLE
Use stdint.h for dwords, etc

### PR DESCRIPTION
**Pull request by dlbeswick**

### CODE REVIEW

File: src/lib/bml/BuzzMachineLoader/MachineInterface.h [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 #include <string.h>
```
Comment (ensonic) [inline src/lib/bml/BuzzMachineLoader/MachineInterface.h]: add a 
#include <stdint.h>
here too?

**Comment by ensonic** [comment]
Could you squash the commits into 1 and force push to the PR branch. Let me know if you need help.

**Comment by dlbeswick** [comment]
Thanks Stefan, I'll be sure to prepare my branches like that in future. 

I found a few other cases, I put the typedefs into a header file to tidy it up.

I also found a case in the windef.h file that seemed like it needed fixing up.

File: src/lib/types.h [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#ifndef LIB_TYPES_H
```
Comment (ensonic) [inline src/lib/types.h]: this is a new file now and would need to be covered by Makefile.am. I think what you previously did was great. Just include stdint.h and have the typedefs adjusted.

**Comment by dlbeswick** [comment]
Sure, I'll change it back.

**Pull request merged**